src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -447,5 +509,15 @@ private List<NamedPropertyExpression> CreateGroupByMemberAssignments(IEnumerable
 
             return properties;
         }
+
+        private static Expression WrapDynamicCastIfNeeded(Expression propertyAccessor)
```

Comment:
>rivate static Ex [](start = 9, length = 16)

not sure why we've turned off the style cop rule requiring static methods before instance.